Backport for linuxthreads.new from:
   http://cgit.uclibc-ng.org/cgi/cgit/uclibc-ng.git/commit/?id=543308f6c46cf2edf8a524bc9c631e472570fe72

	linuxthreads: add back signal.h

	Somehow this got removed with f1d7505e40654a185843bdc8f1cf1fd00ab55c04.

	Reported-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

diff -Naur a/libpthread/linuxthreads/sysdeps/pthread/pthread.h b/libpthread/linuxthreads/sysdeps/pthread/pthread.h
--- a/libpthread/linuxthreads/sysdeps/pthread/pthread.h	2020-09-22 16:38:48.000000000 +0200
+++ b/libpthread/linuxthreads/sysdeps/pthread/pthread.h	2020-09-22 18:32:09.402000132 +0200
@@ -20,6 +20,7 @@
 #include <sched.h>
 #include <time.h>
 
+#include <signal.h>
 #include <bits/pthreadtypes.h>
 #include <bits/initspin.h>
 
